headers_test.go

```diff
@@ -162,6 +162,34 @@ func TestProtectedHeader_MarshalCBOR(t *testing.T) {
 			},
 			wantErr: "protected header: header parameter: content type: require tstr / uint type",
 		},
+		{
+			name: "content type empty",
+			h: ProtectedHeader{
+				HeaderLabelContentType: "",
+			},
+			wantErr: "protected header: header parameter: content type: require non-empty string",
+		},
+		{
+			name: "content type leading space",
+			h: ProtectedHeader{
+				HeaderLabelContentType: " a/b",
+			},
+			wantErr: "protected header: header parameter: content type: require no leading/trailing whitespace",
+		},
+		{
+			name: "content type trailing space",
+			h: ProtectedHeader{
+				HeaderLabelContentType: "a/b ",
+			},
+			wantErr: "protected header: header parameter: content type: require no leading/trailing whitespace",
+		},
+		{
+			name: "content type trailing space",
+			h: ProtectedHeader{
+				HeaderLabelContentType: "ab",
+			},
+			wantErr: "protected header: header parameter: content type: require text of form type/subtype",
+		},
```

Comment:
Do you also intend to add a test to check for multiple `/`

Reply:
Good point. Will do.